--- /Volumes/Data/Users/xdever/asd/Src/main.c	2015-12-05 16:59:39.000000000 +0100
+++ src/STM32/Src/main.c	2015-12-05 16:54:45.000000000 +0100
@@ -60,7 +60,7 @@
 
 /* USER CODE END 0 */
 
-int main(void)
+void init_HW(void)
 {
 
   /* USER CODE BEGIN 1 */
@@ -84,16 +84,6 @@
 
   /* USER CODE END 2 */
 
-  /* Infinite loop */
-  /* USER CODE BEGIN WHILE */
-  while (1)
-  {
-  /* USER CODE END WHILE */
-
-  /* USER CODE BEGIN 3 */
-
-  }
-  /* USER CODE END 3 */
 
 }
 
